examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.matter

```diff
@@ -3001,14 +3002,14 @@ server cluster ThreadNetworkDiagnostics = 53 {
   readonly attribute nullable octet_string<17> meshLocalPrefix = 5;
   readonly attribute NeighborTable neighborTableList[] = 7;
   readonly attribute RouteTable routeTableList[] = 8;
-  readonly attribute int32u partitionId = 9;
-  readonly attribute int8u weighting = 10;
-  readonly attribute int8u dataVersion = 11;
-  readonly attribute int8u stableDataVersion = 12;
-  readonly attribute int8u leaderRouterId = 13;
-  readonly attribute SecurityPolicy securityPolicy[] = 59;
-  readonly attribute octet_string<4> channelMask = 60;
-  readonly attribute OperationalDatasetComponents operationalDatasetComponents[] = 61;
+  readonly attribute nullable int32u partitionId = 9;
+  readonly attribute nullable int8u weighting = 10;
+  readonly attribute nullable int8u dataVersion = 11;
```

Comment:
Please regen ZAP.

Also please add `TestEventTrigger` stuff to the all-clusters-minimal-app.matter